### DISCUSSION TITLE
Please stop the bot spam

### DISCUSSION THREAD

**Discussion by remram44** [python-poetry/poetry#9092] [General] [answered]
### Description

Every time your `github-actions[bot]` locks A CLOSED ISSUE for inactivity, it first posts a comment. That means I get a new email thread for every single one. Those are issues that have been closed for a very long time.

Useless unwanted emails are the definition of spam. Please make it stop.

![Screenshot_20240302_011151](https://github.com/python-poetry/poetry/assets/426784/fbb54597-3bae-4ff1-82c3-5a1a667804e6)

![Screenshot_20240302_011131](https://github.com/python-poetry/poetry/assets/426784/6d51b043-2863-4227-8c35-4805d4fd851e)

### Workarounds

None known

### Poetry Installation Method

install.python-poetry.org

### Operating System

Ubuntu

### Poetry Version

1.6.1

### Poetry Configuration

```bash session
-
```

### Python Sysconfig

```bash session
-
```

### Example pyproject.toml

```TOML
-
```

### Poetry Runtime Logs

```bash session
-
```

**Comment by neersighted** [comment, marked as answer]
This is a one-time cleanup process designed to prevent future frustration with comments that create unwanted notifications (due to being off-topic/describing an unrelated issue) or are hard to discover and valuable (in the case of a bug that is not fully fixed).

This only affects closed issues, and believe me, the maintainers are painfully aware of how many notifications are triggered. It will be done soon, and future notifications generated by this repository should generally be much higher signal-to-noise.

**Reply by remram44** [reply]
Can't you just lock without adding the comment? I don't seem to get a notification of the locking. Is the comment really needed?

**Reply by neersighted** [reply]
The lock triggers a state-change notification, in my experience with this on other repos. Given a notification, it seems better to comment so that users can understand why the lock was applied (and those stumbling in from Google know they are welcome to to open a new issue).

**Comment by remram44** [comment]
I found a workaround: you can go here https://github.com/notifications/subscriptions?repository=MDEwOlJlcG9zaXRvcnkxMjMzMDM0MDI%3D and unsubscribe from the closed issues

**Comment by jrabbit** [comment]
It's pretty hopelessly hostile to do this. My goodwill is irreparably harmed.
